packages/terra-collapsible-menu-view/src/CollapsibleMenuViewHyperlink.jsx

```diff
@@ -110,6 +110,7 @@ const CollapsibleMenuViewHyperlink = ({
   return (
     <div className={cx(['face-up-item'])}>
       <Hyperlink
+        {...customProps}
```

Comment:
Why did we have to add custom props to this component?

Reply:
`key` was not being passed in when using hyperlink from the filepath level. So this was added for that

Reply:
Can we not just update to pass the key directly?

Reply:
We could yes, but other custom props such as `id` might be needed in certain situations too. Plus this has been the pattern for all our terra components as well, I just forgot to add it with the last PR